cmd/occollector/app/collector/collector.go

```diff
@@ -0,0 +1,160 @@
+// Copyright 2018, OpenCensus Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package collector handles the command-line, configuration, and runs the OC collector.
+package collector
+
+import (
+	"log"
+	"os"
+	"os/signal"
+	"strings"
+	"syscall"
+
+	"github.com/census-instrumentation/opencensus-service/internal/collector/jaeger"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/opencensus"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/processor"
+	"github.com/census-instrumentation/opencensus-service/internal/collector/zipkin"
```

Comment:
It'd be nice to separate the OpenCensus imports from the others i.e. please place the others before the opencensus* imports
```go
   "github.com/spf13/cobra"
   "github.com/spf13/viper"
   "go.uber.org/zap"
   "go.uber.org/zap/zapcore"

   "github.com/census-instrumentation/opencensus-service/internal/collector/jaeger"
   ....
)
```